include/pacemaker.h

```diff
@@ -186,6 +186,20 @@ int pcmk_list_nodes(xmlNodePtr *xml, char *node_types);
  */
 int pcmk_status(xmlNodePtr *xml);
 
+/*!
+ * \brief Check whether each rule in a list is in effect
+ *
+ * \param[in,out] xml       The destination for the result, as an XML tree
+ * \param[in]     input     The CIB XML to check (if \c NULL, use current CIB)
+ * \param[in]     date      Check whether the rule is in effect at this date and
+ *                          time (if \c NULL, use current date and time)
+ * \param[in]     rule_ids  The IDs of the rules to check
+ *
+ * \return Standard Pacemaker return code
+ */
+int pcmk_check_rules(xmlNodePtr *xml, xmlNodePtr input, crm_time_t *date,
```

Comment:
Not sure why these are getting documented in the header file when we've typically put doxygen comments in the code, but I went ahead and followed the pattern.

Reply:
Either is acceptable. Documenting in the header is easier for external developers who might think to look to the headers for documentation rather than the online API docs; documenting in the source is easier for us when reading code.